pkg/services/sqlstore/session_test.go

```diff
@@ -44,6 +44,19 @@ func TestRetryingDisabled(t *testing.T) {
 			require.Equal(t, 1, i)
 		})
 
+		t.Run(fmt.Sprintf("%s should return the sqlite3.Error immediately", name), func(t *testing.T) {
```

Comment:
Nit: Might be nice to change `sqlite3.Error` to `sqlite3.ErrBusy`/`sqlite3.ErrLocked` depending on the test case.